src/SFA.DAS.IdentifyDataLocks.IntegrationTests/LearnerWithStoppedApprenticeship/LearnerWithStoppedApprenticeship.cs

```diff
@@ -0,0 +1,38 @@
+using FluentAssertions;
+using Newtonsoft.Json;
+using NSubstitute;
+using NUnit.Framework;
+using SFA.DAS.IdentifyDataLocks.IntegrationTests.Helpers;
+using SFA.DAS.Payments.Model.Core.Audit;
+using SFA.DAS.Payments.Model.Core.Entities;
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.IdentifyDataLocks.IntegrationTests.LearnerWithStoppedApprenticeship
+{
+    public class LearnerWithStoppedApprenticeship : WebApplicationTestFixture
+    {
+        [Test]
+        public async Task RendersDataMismatches()
+        {
+            await Arrange("SFA.DAS.IdentifyDataLocks.IntegrationTests.TestData.LearnerWithStoppedApprenticeship");
+
+            var result = await CreateClient().GetAsync("/Learner/6279465119");
+
+            result.EnsureSuccessStatusCode(); // Status Code 200-299
+            result.Content.Headers.ContentType.ToString().Should().Be("text/html; charset=utf-8");
+        }
+
+        [Test]
+        public async Task RendersDataMismatches2()
```

Comment:
Both the methods look very similar, unless I am missing something.

Reply:
Yep, looks like copy-n-paste-n-forget :)